src/modules/FioAddress/action.js

```diff
@@ -8,9 +8,17 @@ import s from '../../locales/strings'
 import type { Dispatch, GetState } from '../../types/reduxTypes'
 import { getAccount } from '../Core/selectors'
 import { getExchangeDenomination } from '../Settings/selectors'
-import { getFioWallets } from '../UI/selectors'
+import { findWalletByFioAddress, getFioWallets } from '../UI/selectors'
 import type { BuyAddressResponse } from './reducer'
 
+export const setFioWalletByFioAddress = (fioAddressToUse: string) => async (dispatch: Dispatch, getState: GetState) => {
```

Comment:
More unnecessary redux. This should just be state passed from one scenen to another

Reply:
we using it in next functionality. also it's better searching for wallet in actions not in the component

Reply:
refactored